_sass/mixins/_type-mixins.scss

```diff
@@ -11,39 +11,45 @@
   font-size: $base-font-size;
 
   @if $level == h1 {
-    // bold 30/40
-    font-size: 1.875rem; // 30px
+    // bold 38/48
+    color: $black;
+    font-size: 2rem; // 38px
     font-weight: $weight-bold;
-    line-height: 2.5rem; // 40px
+    line-height: 2.5rem; // 48px
   }
 
   @if $level == h2 {
-    // light 32/40
-    font-size: 2rem; // 32px
-    font-weight: $weight-light;
-    line-height: 2.5rem; // 40px
+    // light 30/38.4
+    color: $gray-darkest;
```

Comment:
We're changing the color for h2s globally? Does that work on all of the other pages?